dali/core/mm/monotonic_resource_test.cc

```diff
@@ -69,25 +69,26 @@ TEST(MMTest, MonotonicHostResource) {
 }
 
 TEST(MMTest, MonotonicDeviceResource) {
-  test_host_resource upstream;
+  test_device_resource upstream;
```

Comment:
So RMM is not applicable to the host memory yet, or there is another reason for this change?

Reply:
Previously the resources did not distinguish memory type.

Reply:
Understood